### PR TITLE
docs: slack & fixed broken links

### LINKED ISSUE #2492: docs: update readme
**Summary:**

The readme of our project is outdated and hence we need to update this to meet our current standards

### PR DESCRIPTION
**Pull request by PratyushSingh07**
Fixes #2492 

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,3 +90,16 @@ For Payment Hub usecases, check this [documentation](https://mifos.gitbook.io/do
 ## Note
 
 The UI design is currently being revamped. New design can be found [here](https://docs.google.com/presentation/d/1yFR19vGlKW-amxzGms8TgPzd1jWkrALPFcaC85EyYpw/edit#slide=id.g6c6ccd991d_0_42)
+
+## Join Us on Slack
+
+Mifos boasts an active and vibrant contributor community, which you can engage with [here](https://mifos.slack.com/). Once you've signed in, please explore the various channels, and make sure to join the `#mifos-mobile` channel. This channel encompasses all members involved with this specific project.
```
Comment (therajanmaurya) [inline README.md]: Please update the text with 

Mifos boasts an active and vibrant contributor community, Please join us on slack https://mifos.slack.com/ Once you've joined the mifos slack community, please join the `#mifos-mobile` channel to engage with mifos-mobile development.

File: README.md [hunk 2]
```diff
@@ -90,3 +90,16 @@ For Payment Hub usecases, check this [documentation](https://mifos.gitbook.io/do
 ## Note
 
 The UI design is currently being revamped. New design can be found [here](https://docs.google.com/presentation/d/1yFR19vGlKW-amxzGms8TgPzd1jWkrALPFcaC85EyYpw/edit#slide=id.g6c6ccd991d_0_42)
+
+## Join Us on Slack
```
Comment (therajanmaurya) [inline README.md:94]: Also add in this table https://github.com/openMF/mifos-mobile/blob/development/README.md?plain=1#L9

**Review by therajanmaurya — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (PratyushSingh07) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (PratyushSingh07) [reply README.md:94]: done

**Pull request merged**